src/bitv.c

```diff
@@ -480,21 +487,23 @@ bitv_free_marked (bitv_cb_t * cb, bitv_word * bvec, bitv_word * endvec,
   if (endmrk && !(bgnmrk & bit_zero))
     {
       int units = __builtin_clzl (endmrk);
-      bitv_word chkmsk;
 
-      alloc_units = units + 1;
```

Comment:
Don't need to, I was reorganizing the code and forgot this change behind but it doesn't make difference.